ui/src/app/views/application/show/workflow/tree/item/application.workflow.item.component.ts

```diff
@@ -288,4 +290,35 @@ export class ApplicationWorkflowItemComponent {
     getTriggerSource(pb: PipelineBuild): string {
         return PipelineBuild.GetTriggerSource(pb);
     }
+
+    handleNotification(pipeline: Pipeline): void {
+        switch (pipeline.last_pipeline_build.status) {
+        case 'Success':
+            this._notification.create(this._translate.instant('notification_on_pipeline_success', {
+                pipelineName: pipeline.name
+            })).subscribe();
```

Comment:
no need to destroy subscription on the onDestroy method of the component? ( for this one and all others)